Makefile

```diff
@@ -580,40 +580,6 @@ integration-tests: cmd/mimir/$(UPTODATE)
 web-serve:
 	cd website && hugo --config config.toml --minify -v server
 
-# Generate binaries for a Mimir release
-dist: dist/$(UPTODATE)
-
-dist/$(UPTODATE):
-	rm -fr ./dist
-	mkdir -p ./dist
-	# Build binaries for various architectures and operating systems. Only
-	# mimirtool supports Windows for now. Also darwin/386 is not a valid
-	# architecture.
-	for os in linux darwin windows; do \
-		for arch in 386 amd64 arm64; do \
-			suffix="" ; \
-			if [ "$$os" = "windows" ]; then \
-				suffix=".exe" ; \
-			fi; \
-			if [ "$$os" = "darwin" ] && [ "$$arch" = "386" ]; then \
-				continue; \
-			fi; \
```

Comment:
Excluding darwin/386 is not in the other one

Reply:
Full diff:
```
***************
*** 2,14 ****
  	rm -fr ./dist
  	mkdir -p ./dist
  	# Build binaries for various architectures and operating systems. Only
! 	# mimirtool supports Windows for now.
  	for os in linux darwin windows; do \
! 		for arch in amd64 arm64; do \
  			suffix="" ; \
  			if [ "$$os" = "windows" ]; then \
  				suffix=".exe" ; \
  			fi; \
  			echo "Building mimirtool for $$os/$$arch"; \
  			GOOS=$$os GOARCH=$$arch CGO_ENABLED=0 go build $(GO_FLAGS) -o ./dist/mimirtool-$$os-$$arch$$suffix ./cmd/mimirtool; \
  			sha256sum ./dist/mimirtool-$$os-$$arch$$suffix | cut -d ' ' -f 1 > ./dist/mimirtool-$$os-$$arch$$suffix-sha-256; \
--- 2,18 ----
  	rm -fr ./dist
  	mkdir -p ./dist
  	# Build binaries for various architectures and operating systems. Only
! 	# mimirtool supports Windows for now. Also darwin/386 is not a valid
! 	# architecture.
  	for os in linux darwin windows; do \
! 		for arch in 386 amd64 arm64; do \
  			suffix="" ; \
  			if [ "$$os" = "windows" ]; then \
  				suffix=".exe" ; \
  			fi; \
+ 			if [ "$$os" = "darwin" ] && [ "$$arch" = "386" ]; then \
+ 				continue; \
+ 			fi; \
  			echo "Building mimirtool for $$os/$$arch"; \
  			GOOS=$$os GOARCH=$$arch CGO_ENABLED=0 go build $(GO_FLAGS) -o ./dist/mimirtool-$$os-$$arch$$suffix ./cmd/mimirtool; \
  			sha256sum ./dist/mimirtool-$$os-$$arch$$suffix | cut -d ' ' -f 1 > ./dist/mimirtool-$$os-$$arch$$suffix-sha-256; \
***************
*** 21,32 ****
  			echo "Building query-tee for $$os/$$arch"; \
  			GOOS=$$os GOARCH=$$arch CGO_ENABLED=0 go build $(GO_FLAGS) -o ./dist/query-tee-$$os-$$arch$$suffix ./cmd/query-tee; \
  			sha256sum ./dist/query-tee-$$os-$$arch$$suffix | cut -d ' ' -f 1 > ./dist/query-tee-$$os-$$arch$$suffix-sha-256; \
- 			echo "Building metaconvert for $$os/$$arch"; \
- 			GOOS=$$os GOARCH=$$arch CGO_ENABLED=0 go build $(GO_FLAGS) -o ./dist/metaconvert-$$os-$$arch$$suffix ./cmd/metaconvert; \
- 			sha256sum ./dist/metaconvert-$$os-$$arch$$suffix | cut -d ' ' -f 1 > ./dist/metaconvert-$$os-$$arch$$suffix-sha-256; \
- 			echo "Building mimir-continuous-test for $$os/$$arch"; \
- 			GOOS=$$os GOARCH=$$arch CGO_ENABLED=0 go build $(GO_FLAGS) -o ./dist/mimir-continuous-test-$$os-$$arch$$suffix ./cmd/mimir-continuous-test; \
- 			sha256sum ./dist/mimir-continuous-test-$$os-$$arch$$suffix | cut -d ' ' -f 1 > ./dist/mimir-continuous-test-$$os-$$arch$$suffix-sha-256; \
  			done; \
  		done; \
  		touch $@
--- 25,30 ----
```

Reply:
We don't exclude it because [we don't do a 386 build](
https://github.com/grafana/mimir/pull/2887/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R388-R389) there wasn't a 386 build last release as well.